### DISCUSSION TITLE
Feature Request: Battery Percentage

### DISCUSSION THREAD

**Discussion by bitmvk** [flet-dev/flet#5526] [Extensions]
**Please Describe The Feature To Be Added**

Give the battery percentage of the device

**Suggest A Solution**

add a property to page like `page.battery`

**Comment by ndonkoHenri** [comment]
Can you try Plyer: https://github.com/kivy/plyer

**Comment by bitmvk** [comment]
plyer uses the native API's provided by the OS. So it needs pyjnius(for android) which in turn needs to be built using buildozer to work properly

**Comment by ndonkoHenri** [comment]
Package to be implemented: https://pub.dev/packages/battery_plus
